From 5577a60b9b8860a322288856ff2419c191ed080f Mon Sep 17 00:00:00 2001
From: nivkner <nivkner@zoho.com>
Date: Wed, 6 Dec 2017 18:51:46 +0200
Subject: [PATCH] allow build on aarch64 (#42)

This fixes #37.
---
 src/dec128.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/dec128.rs b/src/dec128.rs
index ca88440..6841640 100644
--- a/src/dec128.rs
+++ b/src/dec128.rs
@@ -224,7 +224,7 @@ impl Into<u32> for d128 {
 /// payload is 0.
 impl fmt::Display for d128 {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let mut buf = [0 as i8; 43];
+        let mut buf = [0; 43];
         unsafe {
             decQuadToString(self, buf.as_mut().as_mut_ptr());
             let cstr = CStr::from_ptr(buf.as_ptr());
@@ -244,7 +244,7 @@ impl fmt::Debug for d128 {
 /// exponential notation is used the exponent will be a multiple of 3.
 impl fmt::LowerExp for d128 {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let mut buf = [0 as i8; 43];
+        let mut buf = [0; 43];
         unsafe {
             decQuadToEngString(self, buf.as_mut().as_mut_ptr());
             let cstr = CStr::from_ptr(buf.as_ptr());
-- 
2.19.0.rc1

